.github/workflows/python-app.yml

```diff
@@ -52,7 +52,7 @@ jobs:
         echo "client_payload: ${{ toJson(github.event.client_payload) }}"
         echo "NOW=$(date +'%m-%d %H:%M')" >> $GITHUB_ENV
         echo ${{env.NOW}}
-        pytest -v --rootdir= HQSmokeTests/testCases
+        pytest -v --rootdir= HQSmokeTests/testCases -n 4 --dist=loadfile --reruns 1
```

Comment:
Just curious, if I am not wrong you have added these values in the pytest.ini, then why adding again in the yaml?

Reply:
Yes! So `pytest.ini` holds the project configurations. If these arguments stay in `pytest.ini`, the instances of parallel runs and the number of reruns configured would be applicable to any directory that'd use `pytest` command to run the scripts.

@kbo001

Reply:
Ok. Gotcha!